echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/controllers/WebhooksController.groovy

```diff
@@ -50,10 +50,16 @@ class WebhooksController {
         }
       }
       if (source == 'github') {
-        event.content.hash = postedEvent.after
-        event.content.branch = postedEvent.ref.replace('refs/heads/', '')
-        event.content.repoProject = postedEvent.repository.owner.name
-        event.content.slug = postedEvent.repository.name
+        if (event.content.hook_id) {
+          log.debug("Webook ping received from github hook_id=${event.content.hook_id} repository=${event.content.repository.full_name}")
```

Comment:
If this ping is only sent once when the webhook is first established, I think it would make sense to make this a `log.info` instead. Otherwise, one may get in a situation where they have to modify the default logger settings, and the remove/re-establish the webhook while troubleshooting.